extension/gulpfile.js

```diff
@@ -18,9 +18,7 @@ gulp.task('build-plugin', (done) => {
     cp.execSync(`${mvnw()} clean package`, { cwd: serverDir, stdio: [0, 1, 2] });
     gulp.src(path.join(serverDir, 'com.microsoft.java.test.plugin/target/*.jar'))
         .pipe(gulp.dest('./server'));
-    gulp.src(path.join(serverDir, 'com.microsoft.java.test.runner/target/*.jar'))
-        .pipe(gulp.dest('./server'));
-    gulp.src(path.join(serverDir, 'com.microsoft.java.test.runner.junit5/target/*.jar'))
+    gulp.src(path.join(serverDir, 'com.microsoft.java.test.runner/target/*-jar-with-dependencies.jar'))
```

Comment:
See Line 19, I think `*.jar` should already include `*-jar-with-dependencies.jar`?

Reply:
Sorry, it's in different folder